### ISSUE TITLE
problem with commit 641e2b5

### ISSUE THREAD

**Issue by nmegill** [metamath/set.mm#93]
Something went wrong with some scripts I wrote to assist the df-sbc conversion.  Please don't use this commit or merge with it until I fix it in a day or so.

Norm

**Comment by digama0** [comment]
I moved your commit to a new branch `newsbc`; merge it back in to develop when you are done.

To all contributors which pulled this commit: make sure you have the latest version of develop, which does _not_ contain this commit. If your version of develop contains this commit, you will need to use`git checkout develop`; `git reset origin/develop` to undo this.

**Comment by nmegill** [comment]
Actually I was alarmed by the context lines in the 'discouraged' diff and panicked because it looked like a disaster, and opened the issue immediately to put things on hold, before I went back to look at the details.  Everything is actually OK.  I think we should change the diff to output zero context lines.

However, I didn't see your message and push another small fix that became f8f478a.  Is this going to mess things up?

For some reason I'm not being emailed the messages for this issue.  Is there something I need to set to do that?

**Comment by digama0** [comment]
@nmegill Well, it would have been a problem if we were trying to remove commit 641e from develop, but if you say it is alright to stay then we're fine, I'll just remove `newsbc` branch and everything is business as usual.

As for notifications, check that the email at https://github.com/settings/emails is correct, and https://github.com/settings/notifications says that you are participating via "email" and "web". Does the notification button on the right sidebar say you are subscribed?

**Comment by nmegill** [comment]
OK, the problem was that I didn't have "Include your own updates" set.  It is set now.

Also, the discouraged diff lines for f8f478a are all valid now.  Some are related to the df-sbc change (and this will be happening now and then as I implement the change), and the others are uses of ax1cn, ax1ne0,axaddcl by Saveliy's theorems (and 1 of yours and 1 of Stefan's) that I changed to the ax-* versions.

Should we automatically re-create the discouraged file after each Travis run?  The point is to detect changes, which it is doing, and I assume people will look at the Travis output when there is a red X and take corrective action if it is needed.  Otherwise we're going to have valid diffs that propagate on every merge and people will tend to start ignoring them.

**Comment by digama0** [comment]
Generally speaking, the discouraged file will be updated fairly commonly, since for example I have set up mmj2 to automatically keep the discouraged file up to date in my commits. This means that it is possible that a change to the discouraged file corresponds to a proof change from a previous commit by a committer that did not bother to update the file, which is not perfect, but the two changes will still be fairly close together in the commit history, and it won't be hard to track down the offending theorem and fix it in any case. Indeed I also noticed the change to the discouraged file in your last commit and went to go remove ax1cn from Saveliy's theorems (before you beat me to it), so I would say that this was a successful test of the discouraged file mechanism.
